Tests/Shared.Specs/TestingTimer.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions.Common;
+
+namespace FluentAssertions.Specs
+{
+    /// <summary>
+    /// Implementation of <see cref="ITimer"/> for testing purposes only.
+    /// </summary>
+    public class TestingTimer : ITimer
```

Comment:
Can be made `internal`, right?

Reply:
I have changed this.
But is it really useful? This class is defined in the assembly for tests an will never get public.

Reply:
If you keep it public, it implictly becomes part of our "contract". I'd like to avoid that.